MediaBrowser.Model/Dto/MediaSourceInfo.cs

```diff
@@ -70,6 +70,8 @@ public MediaSourceInfo()
 
         public bool IsInfiniteStream { get; set; }
 
+        public bool UseMostCompatibleTranscodingProfile { get; set; }
```

Comment:
I believe new non-nullable properties that are returned via the API need to have `[DefaultValue()]` set

@nielsvanvelzen

Reply:
Now defaults to false